src/test/java/org/folio/services/invoice/InvoiceCancelServiceTest.java

```diff
@@ -231,6 +243,25 @@ public void errorUnreleasingEncumbrances(VertxTestContext vertxTestContext) thro
       });
   }
 
+  @Test
+  public void updatePaymentStatusTest(VertxTestContext vertxTestContext) throws IOException {
+    Invoice invoice = getMockAs(PAID_INVOICE_SAMPLE_PATH, Invoice.class);
+    invoice.setFiscalYearId("0fc631d6-45fb-494d-8b48-321f7a2fccf6");
+    List<InvoiceLine> invoiceLines = List.of(
+      new InvoiceLine().withId("1077f1a2-9c4d-4ff0-9152-adb5646d2107").withPoLineId("f9c7a38d-3f8e-4758-b303-a12d365fbf57"),
+      new InvoiceLine().withId("736a8649-1f7b-45cb-92d9-8aa83a169e9f").withPoLineId("0818c22e-d469-431a-9277-b6849a39e523"),
+      new InvoiceLine().withId("e6666187-e626-432d-87bc-a4bd4b697679").withPoLineId("a36d30c0-6031-4ae8-889b-9408d98b7ee4")
+    );
+
+    setupRestCalls(invoice, false, false);
+    setupPaymentStatusCalls(invoiceLines);
+
+    Future<Void> future = cancelService.cancelInvoice(invoice, invoiceLines, requestContext);
```

Comment:
would be useful to add mockito verify statement to check that orderLineService.updateCompositePoLines is invoked

Reply:
Added verify (checking the related rest call).